@stellar/typescript-wallet-sdk/src/walletSdk/Server/index.ts

```diff
@@ -58,3 +66,31 @@ export const signChallengeTransaction = async ({
     networkPassphrase,
   };
 };
+
+/**
+ * Helper method for parsing a JSON string into an AnchorTransaction.
+ * @param {string} transaction - The json string of an anchor transaction.
+ * @returns {AnchorTransaction} The transaction object.
+ */
+export const parseAnchorTransaction = (
+  transaction: string,
+): AnchorTransaction => {
+  const parsed = JSON.parse(transaction);
+  if (
+    "withdraw_memo_type" in parsed ||
```

Comment:
I think it's better to check `kind` instead (either withdrawal or deposit)

Reply:
that is way better thanks
https://github.com/stellar/typescript-wallet-sdk/pull/111/commits/4acb6305f75d0ae2f74bc44d31d091c5cdf7a09b